server/control-panel.md

```diff
@@ -1,69 +1,39 @@
 # Control Panels
 
-## cPanel
+This document describes how to install WordPress® in the cPanel & WHM and Plesk control panels using WP Toolkit.
 
-This tutorial provides step-by-step examples of creating a database and database user using the cPanel MySQL Database Wizard tool.
+## WP Toolkit 
```

Comment:
Please change back to cPanel or "cPanel WP Toolkit". if you want to cover both Plesk and cPanel, use only one headline like "WP Toolkit for cPanel and Plesk"